modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractDartCodegen.java

```diff
@@ -51,9 +52,7 @@
     protected String apiTestPath = "test" + File.separator;
     protected String modelTestPath = "test" + File.separator;
 
-    // Names that must not be used as model names because they clash with existing
-    // default imports (dart:io, dart:async, package:http etc.) but are not basic dataTypes.
-    protected Set<String> additionalReservedWords;
+    protected Map<String, String> imports = Maps.newHashMap();
```

Comment:
`new HashMap<>();` no need to use guava for this

Reply:
I can change that tomorrow.